fixtures/oauth2_token.json

```diff
@@ -0,0 +1,15 @@
+{
+    "access_token": "91763b24c43d3e344f424e8b|880b55a08c758e87ff8755a00c6b8a12",
```

Comment:
I hope this is not one of your personal token

Reply:
Hehe, don't worry, but thanks for thinking about it.